tictactoe.Tests/GameTest.cs

```diff
@@ -85,5 +83,117 @@ public void GetCurrentPlayerReturnsPlayer1Test()
 
             Assert.Equal("X", currentPlayer._marker);
         }
+
+        [Fact]
+        public void GamePlayChecksForValidMoveTest1()
+        {
+            board.cells = new string[] {
+                "X", "O", " ",
+                " ", " ", " ",
+                " ", " ", " "};
+
+            console.setUserInputs(new List<string> { "1", "1", "2", "2", "3", "4", "5", "6", "7", "8", "9" });
+
+            game = new Game(console, rules);
+
+            game.SetBoard(board);
+            game.SetPlayers(player1, player2);
+
+            game.Menu();
+
+            Assert.Equal("O", board.cells[1]);
+        }
+
+        [Fact]
+        public void GamePlayChecksForValidMoveTest2()
```

Comment:
I would use more descriptive test name than Test1 and Test2. What sets Test1 and Test2 apart?

Reply:
commit 6cfdcbf8f0fbc49d7028d5432427dc6c62641347